server/app.ts

```diff
@@ -28,5 +28,6 @@ app.use('/past-rides', pastride);
 app.use('/vehicles', vehicle);
 app.use('/locations', location);
 app.use('/auth', auth);
+app.get('/health-check', (_, response) => response.status(200).send('OK'));
```

Comment:
This is not just for testing heroku deployment. It's also helpful for debugging in the future.